docs/deployment/helm/README.md

```diff
@@ -0,0 +1,172 @@
+---
+Title: Deploying with Helm
+---
+
+# Deploying with Helm 
+Activiti Enterprise can be deployed using [Helm](https://helm.sh) charts. 
+
+## Prerequisites 
+A Helm deployment assumes that you have the following:
+
+* [Quay.io](https://quay.io) credentials to access Activiti Enterprise images. 
+* A valid license for Activiti Enterprise.
+* A Kubernetes cluster.
+* A Docker registry for the [deployment service](../architecture/platform.md#deployment-service).
+* Helm installed in the cluster. 
```

Comment:
we have to tell that we support only the helm version as in the DBP guidelines that is 2.14.3 the latest is 3.x which does not work https://github.com/Alfresco/alfresco-dbp-deployment/blob/master/README-prerequisite.md

Reply:
Added a version in the prerequisite